src/gcodeExport.cpp

```diff
@@ -197,6 +201,7 @@ std::string GCodeExport::getFileHeader(const double* print_time, const std::vect
         else if (flavor == EGCodeFlavor::REPRAP)
         {
             prefix << ";Filament used: " << ((filament_used.size() >= 1)? filament_used[0] / (1000 * extruder_attr[0].filament_area) : 0) << "m" << new_line;
+            prefix << ";Layer height: " << layer_height << new_line;
```

Comment:
we can make it local to that file (static) and also change the name to something like gcode_comment_layer_height. How about that?